chromadb/server/fastapi/__init__.py

```diff
@@ -507,7 +504,7 @@ async def list_collections(
         if maybe_database:
             database = maybe_database
 
-        return cast(
+        api_collections = cast(
```

Comment:
Weird that this goes `Collection` -> `CollectionModel`

Reply:
We can fix this when we change the underlying API surface to return a Model instead. Which is what I presume you were going to do in your followup that needs this.